### ISSUE TITLE
Suggestion: Notifications

### ISSUE THREAD

**Issue by drkui** [xs9627/feedpop#137]
My suggestion is to add an optional setting to enable toast notifications. That would make it more apparent when there are updates vs checking when/if the number gets updated on the extension.

**Comment by xs9627** [comment]
Good point, will try to add this in next release. Thanks for you feedback!

**Comment by xs9627** [comment]
This feature will be included in v1.2.3. May take 2~3 days before the submit be approved in Chrome web store.
